test/controllers/content_navigation_ab_testing_content_items_controller_test.rb

```diff
@@ -94,5 +63,52 @@ class ContentItemsControllerTest < ActionController::TestCase
         refute @controller.universal_navigation_with_mainstream_nav?
       end
     end
+
+    test "#{document_type} shows no navigation elements for universal no navigation variant" do
+      path = "government/abtest/#{schema_name}"
+      content_item_inside_of_test(document_type, schema_name, path)
+
+      with_variant ContentNavigation: ContentItemsController::CONTENT_NAVIGATION_UNIVERSAL_NO_NAVIGATION do
+        get :show, params: { path: path }
+        refute_partial('shared/_breadcrumbs')
+        refute_partial('shared/_related_items')
+        refute_partial('govuk_component/metadata')
+        refute_partial('govuk_component/document_footer')
+
+        if document_type.in? %w{detailed_guide statutory_guidance}
+          assert_template 'components/_published-dates'
```

Comment:
Do we need to check for the new metadata component too?

Reply:
Yep we should, I will add